stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!--Command Suggestions-->
+    <string name="stream_commands_instant_commands">Instant Commands</string>
+    <string name="stream_command_command_template">/%s %s</string>
+
+    <!--Mention Suggestions-->
+    <string name="stream_mention_user_name_template">/%s</string>
```

Comment:
This string should start with `@` instead of `/`

Reply:
Done, changed the starting character to `@`